pages.es/osx/yabai.md

```diff
@@ -0,0 +1,24 @@
+# yabai
+
+> Un administrador de ventanas en mosaico para macOS basado en la partición de espacio binario.
+> Más información: <https://github.com/koekeishiya/yabai>.
+
+- Establecer el diseño a bsp:
```

Comment:
For Romance languages, the present indicative fits better.
```suggestion
- Establece la diposición a bsp:
```
Not sure what this command does, but for layour sometimes `disposición` (like "la disposición de los objetos en la sala") fits better